LR_Tourist/BLL/Interfaces/ITravelVoucher.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using BLL.Model;
+namespace BLL.Interfaces
+{
+    public interface ITravelVoucher
+    {
+        void MakeOrder(int idTravelVoucher, int idUser );
+
+        TravelVoucherDTO GetTravelVoucher(int? id);
```

Comment:
I guess Id parameter shouldn't be nullable.

Reply:
fixed